lesson-02/hw2/example-04.js

```diff
@@ -0,0 +1,30 @@
+const formatString = function(string) {
+  let res;
+  if (string.length <= 40) {
```

Comment:
В даному випадку не потрібен if...else, можна спростити функцію, використавши тільки одну умову

```suggestion
  if (string.length > 40) {
     const CUTSTRING = string.slice(0, 40);
     return `${CUTSTRING}...`;
  }
  return string;
```